h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2269,34 +2269,34 @@ public void onCompletion(CountedCompleter caller) {
           _state.updateState(beta, gginfo);
         if (!_parms._lambda_search)
           updateProgress(true);
-        boolean converged = _state.converged();
+        boolean converged = _earlyStopEnabled ?false:_state.converged(); // GLM specific early stop.  Disable is early stop is enabled
```

Comment:
could be simplified to `_earlyStopEnabled && _state.converged()`
or made more explicit by `_earlyStopEnabled ? _earlyStop : _state.converged()` and removing `&& ! _earlyStop` from L#2274

also typo in the comment `Disable is early...`should be `Disable if early...`

Reply:
Fixed the type.  Good catch on the simplification